public/index.html

```diff
@@ -2,16 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link href="https://static.inovex.de/css/raleway.css" rel="stylesheet">
+    <link href="https://static.inovex.de/css/raleway.css" rel="stylesheet" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Open source online retrospective & collaboration tool - fast onboarding, jump right into a session with your team"
-    />
+    <meta name="description" content="Open source online retrospective & collaboration tool - fast onboarding, jump right into a session with your team" />
     <meta name="keywords" content="Collaboration, Scrum, Retrospective, Retro, Agile" />
     <title>scrumlr.io - Online collaboration</title>
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
I think we should support touch icons for different devices and resolutions here. See https://www.web-design-und-so.de/webdesign/touch-icons-apple-android-windows/ this link for more information on this topic. Basically a complete list would look something like this:

```
<link rel="shortcut icon" href="%PUBLIC_URL%/logo196.png" sizes="196x196">
<link rel="shortcut icon" href="%PUBLIC_URL%/logo32.png" sizes="32x32">
<link rel="shortcut icon" href="%PUBLIC_URL%/logo16.png" sizes="16x16">

<link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon-180.png">
<link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon-167.png" sizes="167x167"> 
<link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon-152.png" sizes="152x152"> 

<meta name="msapplication-square70x70logo" content="%PUBLIC_URL%/logo70.png">
<meta name="msapplication-square150x150logo" content="%PUBLIC_URL%/logo150.png">
<meta name="msapplication-wide310x150logo" content="%PUBLIC_URL%/logo310x150.png">
<meta name="msapplication-square310x310logo" content="%PUBLIC_URL%/logo310.png">
```

I can ask  our design team to provide you with the resources.